Ullaakut/cameradar
# Add default credentials for aiphone cameras

## Goal of this PR

This PR fixes #202 by adding the default credentials and route for aiphone cameras. It also reorders the route dictionary alphabetically.

## How to test it 

* Set up a RTSPATT stream using aiphone default credentials and RTSP URI, and run cameradar on it
* It should be accessed successfully
